android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -771,12 +774,26 @@ public void testFieldsExist() throws Exception {
                 "org/chromium/chrome/browser/ntp/NewTabPageLayout", "mMvTilesContainerLayout"));
         Assert.assertTrue(
                 fieldExists("org/chromium/chrome/browser/dom_distiller/ReaderModeManager", "mTab"));
+
         Assert.assertTrue(fieldExists(
                 "org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder",
                 "mDropdownHeight"));
         Assert.assertTrue(fieldExists(
                 "org/chromium/chrome/browser/omnibox/suggestions/DropdownItemViewInfoListBuilder",
                 "mPriorityOrderedSuggestionProcessors"));
+
+        Assert.assertFalse(fieldExists(
+                "org/chromium/chrome/browser/search_engines/settings/BraveBaseSearchEngineAdapter",
+                "mPrepopulatedSearchEngines"));
+        Assert.assertFalse(fieldExists(
+                "org/chromium/chrome/browser/search_engines/settings/BraveBaseSearchEngineAdapter",
+                "mRecentSearchEngines"));
+        Assert.assertFalse(fieldExists(
+                "org/chromium/chrome/browser/search_engines/settings/BraveBaseSearchEngineAdapter",
+                "mSelectedSearchEnginePosition"));
+        Assert.assertFalse(fieldExists(
```

Comment:
Not sure I understood this empty field check.

Reply:
Sorry, this is an accidental leftover